### PR TITLE
Miecraft?

### LINKED ISSUE #2166: [0.4.11] Diamond Honeycomb has no Centrifuge recipe
**Modpack version**
4.11

**Describe the bug**
You can not centrifuge diamond honeycomb to get anything from it.

**Logs**
Input pastebin link to crash-report or `latest.log`, if applicable.

**Screenshots**

image
What should be there

image
What is there

### PR DESCRIPTION
**Pull request by TemperedFool**
Typing is hard - correcting a few spelling bumbles in Forest/Rocky/Diamond/Soupy bees. Fixes #2166.

### CODE REVIEW

File: config/resourcefulbees/bees/special/Soup.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
         "mutations": [
             {
                 "type": "ITEM",
-                "inputID": "minecraft:bowl",
+                "inputID": "minecraft:oak_log",
```
Comment (NillerMedDild) [inline config/resourcefulbees/bees/special/Soup.json:24]: Is this change intended? The bowl made a bit more sense imo.

Comment (TemperedFool) [reply config/resourcefulbees/bees/special/Soup.json:24]: It totally does, and currently isn't supported unfortunately. That's why the recipe was failing to load - you can mutate TO an item, but not from one. Closest thing to a bowl I could come up with was a block of wood... I would change this if/when Epic implements item mutations.

**Comment by NillerMedDild** [comment]
Thanks!

**Comment by TemperedFool** [comment]
Least I can do is clean up my own mistakes when I'm capable of it...
